# 多个检测到的物体信息

# 标准ROS消息头
std_msgs/Header header

# 检测到的物体数组
DetectedObject[] objects